tests/test_examples/test_configure_hydra.py

```diff
@@ -69,6 +69,19 @@ def test_job_name_with_config_override(tmpdir: Path) -> None:
     assert result == "name_from_config_file"
 
 
+def test_job_override_dirname(tmpdir: Path) -> None:
+    cmd = [
+        "examples/configure_hydra/job_override_dirname/my_app.py",
+        "hydra.sweep.dir=" + str(tmpdir),
+        "a=10",
+        "b=20",
```

Comment:
Do a sweep over multiple runs:
```suggestion
        "a=a1,a2",
        "b=b1",
```